playbook/app/pb_kits/playbook/pb_date_picker/date_picker_helper.js

```diff
@@ -52,9 +53,15 @@ const datePickerHelper = (config) => {
     }
   }
 
-  const setMonthAndYearPlugin = () => (
-    plugins ? [ monthSelectPlugin({ shorthand: true, dateFormat: 'F Y', altFormat: 'F Y' }) ] : []
-  )
+  const setPlugin = () => {
+    let p 
+    if (plugins === "ms") {
```

Comment:
```tsx
    if (selectionType === "month" || plugins === true) {
```

In this very particular case, we would be requiring that a new major release of Playbook be created as now the `plugins` feature has been changed from how it originally worked. I don't think we will want to do this within this unit of work. Instead, what would work best is to introduce the new feature atop the existing feature as you can see in the logic above. Here we are still checking for `plugins === true` thereby keeping the boolean toggle in place to enable only month selection just as it was before.